src/go/rpk/pkg/cli/registry/schema/get.go

```diff
@@ -24,11 +24,12 @@ import (
 
 func newGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	var (
-		deleted    bool
-		id         int
-		schemaFile string
-		schemaType string
-		sversion   string
+		deleted     bool
```

Comment:
Does the order that we put here reflect the order that gets output in the help text? 

Wondering if could/should order flags alphabetically.

Reply:
No, these don't change the flag order in the help text.

However, we should think about whether having the flags ordered alphabetically makes sense, or not. Typically, we order them according to what we think will be the most used options. But I don't think we keep this consistency over time (and as we keep adding more flags to existing commands)